tests/phpunit/elementor/modules/atomic-widgets/test-style-rendering.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+namespace Elementor\Testing\Modules\AtomicWidgets;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use Elementor\Modules\AtomicWidgets\Style\Style_Render;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+    exit; // Exit if accessed directly.
+}
+
+class Test_Style_Rendering extends Elementor_Test_Base {
```

Comment:
```suggestion
class Test_Styles_Renderer extends Elementor_Test_Base {
```

also rename the file name
it should match the tested code

Reply:
Done